src/js/main.js

```diff
@@ -1,9 +1,8 @@
 import productList from "./productList.mjs";
-import { loadHeaderFooter } from "./utils.mjs";
+import { loadHeaderFooter, getParam } from "./utils.mjs";
 import { cartItemCountUpdate } from "./cartImageAdjuster.js";
 
-
-  loadHeaderFooter();
-  productList(".product-list", "tents");
-  cartItemCountUpdate();
-
+loadHeaderFooter();
+const category = getParam("category");
+productList(".product-list", category);
+cartItemCountUpdate();
```

Comment:
We moved this over from the product-list.js so that they didn't have to duplicate and compete with each other.